tests/src/test/scala/apigw/healthtests/ApiGwEndToEndTests.scala

```diff
@@ -142,6 +142,120 @@ class ApiGwEndToEndTests
         }
     }
 
+    it should "return a list of alphabetized api-experimental" in withAssetCleaner(wskprops) {
```

Comment:
Looks like all the API tests can be refactored into one test as they share common code.